sdk/communication/Azure.Communication.Calling.Server/src/ConversationClient.cs

```diff
@@ -0,0 +1,546 @@
+            }
+            catch (Exception ex)
+            {
+                scope.Failed(ex);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Remove participant from the call.
+        /// </summary>
+        /// <param name="conversationId"> The conversation id that can be a group id or a encoded conversation url retrieve from client. </param>
+        /// <param name="participantId">The participant id.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        public virtual async Task<Response> RemoveParticipantAsync(string conversationId, string participantId, CancellationToken cancellationToken = default)
+        {
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(ConversationClient)}.{nameof(RemoveParticipantAsync)}");
+            scope.Start();
+            try
+            {
+                return await RestClient.RemoveParticipantAsync(conversationId, participantId, cancellationToken).ConfigureAwait(false);
+            }
+            catch (Exception ex)
+            {
+                scope.Failed(ex);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Start recording of the call.
+        /// </summary>
+        /// <param name="conversationId"> The conversation id that can be a group id or a encoded conversation url retrieve from client. </param>
+        /// <param name="recordingStateCallbackUri">The uri to send state change callbacks.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        public virtual async Task<Response<StartCallRecordingResponse>> StartRecordingAsync(string conversationId, Uri recordingStateCallbackUri, CancellationToken cancellationToken = default)
+        {
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(ConversationClient)}.{nameof(StartRecordingAsync)}");
+            scope.Start();
+            try
+            {
+                return await RestClient.StartRecordingAsync(conversationId, recordingStateCallbackUri.AbsoluteUri, cancellationToken).ConfigureAwait(false);
+            }
+            catch (Exception ex)
+            {
+                scope.Failed(ex);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Start recording of the call.
+        /// </summary>
+        /// <param name="conversationId"> The conversation id that can be a group id or a encoded conversation url retrieve from client. </param>
+        /// <param name="recordingStateCallbackUri">The uri to send state change callbacks.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        public virtual Response<StartCallRecordingResponse> StartRecording(string conversationId, Uri recordingStateCallbackUri, CancellationToken cancellationToken = default)
```

Comment:
I thought these were supposed to become long running operation.

Reply:
This Operation has no difference with others. So I don't think it's long running operation.